runbooks-placement-strategy/README.md

```diff
@@ -0,0 +1,96 @@
+---
+creation_date: 2025-06-12
+issues:
+- https://github.com/giantswarm/roadmap/issues/2838
+owners:
+- https://github.com/orgs/giantswarm/teams/team-planeteers
+state: review
+summary: This RFC addresses the strategic placement of operational runbooks (currently called "Ops recipes") to improve accessibility, maintainability, and security while ensuring consistent structure and content sanitization.
+---
+
+# Strategic placement and standardization of operational runbooks
+
+## Problem statement
+
+Our operational runbooks (currently referred to as "Ops recipes") are currently scattered across multiple locations, specifically between our private intranet and public handbook. This fragmented approach creates several challenges:
+
+1. **Inconsistent accessibility**: Some runbooks are private when they could be public, limiting knowledge sharing and community contribution
+2. **Lack of standardization**: Runbooks follow different formats and structures across locations
+3. **Security concerns**: Some runbooks may contain sensitive information that needs to be identified and sanitized
+4. **Discovery issues**: Engineers struggle to find the right runbook when they're distributed across different platforms
+5. **Maintenance overhead**: Managing runbooks in multiple locations increases maintenance burden
+
+We need a unified decision for runbook placement that ensures:
+
+- Proper security and data sanitization
+- Consistent structure and format
+- Optimal accessibility for both internal teams and the community
+- Clear governance and maintenance processes
+
+### Decision maker
+
+Planeteers team.
+
+### Deadline
+
+30.07.2025
+
```

Comment:
```suggestion
```

We have the `owners` field for this, and the deadline isn't relevant in a few weeks, so I'd leave this out (and deadlines usually don't work in RFCs).